qqqwww123cdv/hometask-4/my-array-rotate/my_array_rotate_benchmark.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'benchmark'
+require './my_array_rotate'
+
+Benchmark.bm do |x|
+  p 'my lab_rotate'
+  x.report { (1..100).to_a.lab_rotate(10) }
+  x.report { (1..100).to_a.lab_rotate(10) }
+  x.report { (1..100).to_a.lab_rotate(10) }
+  x.report { (1..100).to_a.lab_rotate(10) }
+  x.report { (1..100).to_a.lab_rotate(10) }
```

Comment:
let's use array size increasingly (e.g. from 10 to 10000) to have more decent statistics